src/Desktop/Utility/StandardLinksHelper.cs

```diff
@@ -25,7 +25,7 @@ public class StandardLinksHelper
         StandardLinksHelper()
         {
             // get the path of dictionary file.
-            var json = File.ReadAllText(Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "links.json"));
+            var json = File.ReadAllText(Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Axe.Windows.CodeSampleLinks.json"));
```

Comment:
Changing the file name here creates a runtime break in AIWin's MSI-based install because we'll need to update [Product.wxs](https://github.com/microsoft/accessibility-insights-windows/blob/main/src/MSI/Product.wxs) to get the new file instead of the old one. That's a 1 line change that we'll make when we consume the updated package.

Reply:
The rename makes sense to me! 

@DaveTryon Curiousity question for whenever you have a moment: I'm trying to understand is the value of `Assembly.GetExecutingAssembly().Location`. Since we are replacing `links.json` which was imported from Accessibility Insights for Windows, I'm not piecing together why this location remains the same. My best guess is that during build (or related process) `links.json` and now `Axe.Windows.CodeSampleLinks.json` are moved to the location defined by `Assembly.GetExecutingAssembly().Location` -- is that accurate? Just trying to put the pieces together :)

Reply:
When AIWin consumes the NuGet package, the contents of the package all move together. Since we put the json file in the same folder as the DLL's, it's in the correct location by default. Please ping me if this doesn't answer your question.